dev/TabView/TabView_themeresources.xaml

```diff
@@ -6,7 +6,7 @@
 
     <ResourceDictionary.ThemeDictionaries>
         <ResourceDictionary x:Key="Light">
-            <StaticResource x:Key="TabViewBackground"                        ResourceKey="SystemControlBackgroundBaseLowBrush" />
```

Comment:
was it intentional to change this for Light but not for Dark and HC?

Reply:
Unlikely, especially if you are seeing a bug. Are you able to contribute a fix? @teap is out this week.

Reply:
It was actually intentional. PM, design, and I agreed to make the background slightly lighter for light theme but not darker for dark theme.